### PR TITLE
Solves issue# 1618 (Add placeholder text for the schedule route)

### LINKED ISSUE #1618: Add placeholder text for the schedule route
**Describe the bug**

image
An empty screen is displayed when there is no schedule available for an event

### PR DESCRIPTION
**Pull request by pranav1698**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Previously there was no placeholder for the events with no schedules
Now there is a placeholder for the events

Fixes #1618 
GIF of present changes: 

open-event-frontend

### CODE REVIEW

File: app/templates/public/schedule.hbs [hunk 1]
```diff
@@ -1 +1,7 @@
-{{outlet}}
+
+{{#if outlet}}
+  {{outlet}}
+{{else}}
+  <h4>*Schedule for the event is currently not available</h4>
```
Comment (ritikamotwani) [inline app/templates/public/schedule.hbs]: For the helper text use `<div class="ui disabled header">` instead of the h4 tag.

Comment (pranav1698) [reply app/templates/public/schedule.hbs]: Ok right on it

**Comment by pranav1698** [comment]
Should I remove the if else statements

**Comment by ritikamotwani** [comment]
@pranav1698 yes for now remove it. The if else condition will be added after the scheduler for the event is ready.

File: app/templates/public/schedule.hbs [hunk 2]
```diff
@@ -1 +1 @@
-{{outlet}}
+<div class="ui disabled header">*Schedule for the event is currently not available</div>
```
Comment (ritikamotwani) [inline app/templates/public/schedule.hbs]: Remove the '*' used before Schedule. 
Use 'unavailable' in place of not available.

**Comment by pranav1698** [comment]
@ritikamotwani, are we done with the changes??

**Comment by ritikamotwani** [comment]
@pranav1698 the commits aren't squashed. 
Also the commit message should be something like Add placeholder text for the public schedule route.

**Comment by pranav1698** [comment]
@ritikamotwani  I think It is solved now

**Comment by pranav1698** [comment]
Ya sorry,  recognised that i have that mistake I will take care of it next
time

**Comment by pranav1698** [comment]
@CosmicCoder96 I think this ends it, please review it

**Pull request merged**